src/Microsoft.TestPlatform.ObjectModel/Navigation/DiaSession.cs

```diff
@@ -99,7 +99,7 @@ private static ISymbolReader GetSymbolReader(string binaryPath)
             // For remote scenario, also look for pdb in current directory, (esp for UWP)
             // The alternate search path should be an input from Adapters, but since it is not so currently adding a HACK
             pdbFilePath = !File.Exists(pdbFilePath) ? Path.Combine(Directory.GetCurrentDirectory(), Path.GetFileName(pdbFilePath)) : pdbFilePath;
-            using (var stream = new FileHelper().GetStream(pdbFilePath, FileMode.Open, FileAccess.Read))
+            using (var stream = new FileHelper().GetStream(pdbFilePath, FileMode.Open, FileAccess.Read, FileShare.Read))
```

Comment:
Possibility to have a single method for GetStreamForRead in FileHelper

Reply:
This change not required now. we don't want to add new method which leads to additional learning . The helper is just wrapper to IO apis.